src/packages/axe-config-generator/index.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you look into whether it would be possible for this to just live in `/packages/axe-config-generator` directly, rather than making the separate `/src/packages/axe-config-generator` directory? The UI and report packages use the `/src/` directories for historical reasons - if it's technically feasible, it would be better to just keep the config generation in one place, I think

Reply:
I don't think it's worth spending a ton of time on it, though; if you tried it and it creates difficulties for build/bundler/etc configuration, it's fine to leave it here

Reply:
I know when I created the assessment-validator package recently someone had a similar suggestion and I explored it and found it to be technically problematic... I don't remember all of the details offhand but I believe the basic issue that importing src code into the packages dir causes a lot of complexity with dependencies/building. That's why the package code that stands alone and does not require src imports (like the new index.js added in this PR) lives in top level packages dir whereas the rest of the code lives in src/packages. 

Let me know if you're interested in more details and I can dig a little further to refresh my memory :)